src/main/res/layout/request_permission.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:paddingLeft="16dp"
+    android:paddingRight="16dp">
+
+    <ImageView
+        android:id="@+id/locIcon"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="128dp"
```

Comment:
Have you tried this on a small screen?

Reply:
Yes, the recording in the description was made with a 5" display. Here is the same view in a 3.7" display:

![Location Disclosure 3 4 inch](https://user-images.githubusercontent.com/1608415/102910191-28eeba00-4448-11eb-9340-b79ef02599d4.png)

And here in a large one, 6.7":

![Location Disclosure 6 7 inch](https://user-images.githubusercontent.com/1608415/102910230-36a43f80-4448-11eb-9a96-04d7b915e2ad.jpg)

In all cases the components are displayed a bit different but looks acceptable I think. Is there a better way to define the margin othen than `dp` and other than define the same layout for each screen size? (unfortunately % is not supported in this layout class and other layout classes that support it require API 23 or above).

Taking into account that most of the deployments may use smaller sizes than 5" I can reduce the margin, maybe to 100dp?

Reply:
Reduced to 100dp.

Reply:
Awesome, that's perfect!